ajax/libs/floating.js/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "floating.js",
+  "description": "Float a number of things up on a page (hearts, flowers, ...)",
+  "homepage": "https://haroen.me/floating.js/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Haroenv/floating.js.git"
+  },
+  "author": "Haroen Viaene",
+  "license": "Apache-2.0",
+  "filename": "floating.js",
+  "keywords": [
```

Comment:
I don't think it's worth to care about those two old versions that had another patch just after, right?